src/partials/experymentDetails.handlebars

```diff
@@ -0,0 +1,48 @@
+<div class="row columns">
+    <h4>
+        <a class="expand-button min" for="#{{id}}">
+            Experiment details
+        </a>
+    </h4>
+    <div id="{{id}}" class="box">
+        {{#if model.analysis.experiment_type}}
+            <div class="row">
+                <div class="column small-12 medium-4 large-4 detailList-key">
+                    Experiment type:
+                </div>
+                <div class="column small-12 medium-8 large-8 detailList-value">
+                    {{#replace "_" " "}}
+                        {{model.analysis.experiment_type}}
+                    {{/replace}}
+                </div>
+            </div>
+        {{/if}}
+        {{#if model.runs}}
+            <div class="row">
+                {{#each model.runs}}
+                    {{#if @first }}
+                        <div class="column small-12 medium-4 large-4">
+                            Instruments:
+                        </div>
+                    {{/if}}
+                    <div data-cy="instrument" class="column small-12 medium-8 large-8 {{#unless @first}}small-offset-12 medium-offset-4 large-offset-4{{/unless}}">
+                        platform: {{this.instrument_platform}}, model: {{this.instrument_model}}
```

Comment:
This render a bit weird, e.g:

| Experiment type:  |  assembly |
|-------------------|-----------|
|Instrument model: Illumina HiSeq 2000 | Instrument platform: ILLUMINA |

Sort of:
| KEY:  |  VALUE |
|-------------------|-----------|
|KEY: VALUE | KEY: VALUE |

Reply:
It should be, for hybrids

| Experiment type:  |  assembly |
|-------------------|-----------|
| Intruments:    | model: ILLUMINA, platform: Illumina HiSeq 2000 |
|     | model: PacBio , platform: RS II Instrument |

Reply:
It should be, for hybrids

| Experiment type:  |  assembly |
|-------------------|-----------|
| Intruments:    | model: ILLUMINA, platform: Illumina HiSeq 2000 |
|     | model: PacBio , platform: RS II Instrument |

I'll check this